templates/migration-settings.py.erb

```diff
@@ -1,12 +1,12 @@
 PULP2_MONGODB = {
-    "name": "<%= scope['pulpcore::mongo_db_name'] %>",
-    "seeds": "<%= scope['pulpcore::mongo_db_seeds'] %>",
-    "username": "<%= scope['pulpcore::mongo_db_username'] %>",
-    "password": "<%= scope['pulpcore::mongo_db_password'] %>",
-    "replica_set": "<%= scope['pulpcore::mongo_db_replica_set'] %>",
-    "ssl": "<%= scope['pulpcore::mongo_db_ssl'] %>",
-    "ssl_keyfile": "<%= scope['pulpcore::mongo_db_ssl_keyfile'] %>",
-    "ssl_certfile": "<%= scope['pulpcore::mongo_db_ssl_certfile'] %>",
-    "verify_ssl": "<%= scope['pulpcore::mongo_db_verify_ssl'] %>",
-    "ca_path": "<%= scope['pulpcore::mongo_db_ca_path'] %>",
+    "name": "<%= scope['pulpcore::plugin::migration::mongo_db_name'] %>",
```

Comment:
For future reference, these variables are now in scope so you can use `@mongodb_db_name`